apps/src/code-studio/initApp/project.js

```diff
@@ -1178,6 +1180,13 @@ var projects = (module.exports = {
               if (saveSourcesErrorCount >= NUM_ERRORS_BEFORE_WARNING) {
                 header.showTryAgainDialog();
               }
+              if (err.message.includes('httpStatusCode: 422')) {
+                const msg =
+                  'There was an error saving your project, please remove any invalid characters to resolve.';
+                getStore().dispatch(
+                  displayWorkspaceAlert('error', msg, true) /* bottom */
```

Comment:
nit: you can include the comment inside the parameter list so we know which argument it's referencing, i.e. `displayWorkspaceAlert('error', msg, /* bottom */ true)`

Reply:
Sure - will do!